go-rc.sh

```diff
@@ -46,16 +46,22 @@ log_fatal () {
 }
 
 download () {
-  local repo=$1
-  local jar_path=$2
-  local artifact=$3
+  local jar_path=$1
+  local artifact=$2
   # Set this to anything in the caller to ignore the download error
-  local ignore_err=${4:-}
+  local ignore_err=${3:-}
   local err
   local output
   if [ -f "$jar_path" ]; then
       log_info "$jar_path already exists, skipping download."
   else
+      if [[ "${artifact}" == *-SNAPSHOT ]]
+      then
+        repo=${ENTERPRISE_SNAPSHOT_REPO}
+      else
+        repo=${ENTERPRISE_RELEASE_REPO}
+      fi
```

Comment:
I know this is copied from another part of the script, but just to be more consistent with the `if` above:

```suggestion
      if [[ "${artifact}" == *-SNAPSHOT ]]; then
          repo=${ENTERPRISE_SNAPSHOT_REPO}
      else
          repo=${ENTERPRISE_RELEASE_REPO}
      fi
```